migrations/sql/V2018.02.13.1749__AlterCountryIndex.sql

```diff
@@ -0,0 +1,258 @@
+ALTER TABLE country
+ DROP nid;
+
+ALTER TABLE country
+  ADD nid SMALLINT,
+  ADD CONSTRAINT country_nid_unique UNIQUE(nid);
+
+UPDATE country SET nid=4 WHERE id='AFG';
```

Comment:
Can you remind me why `ALTER TABLE ALTER COLUMN country SET TYPE SMALLINT;` does not work here?

Reply:
or failing that create a new nullable column, copy the data in, set the column non-nullable, and then drop the old one?

Reply:
Those work too, this was just the quickest option

Reply:
Right, just these big piles of numbers make me nervous :)